cmd/openshift-install/testdata/agent/image/configurations/external_compact.txt

```diff
@@ -0,0 +1,43 @@
+# Verify a default configuration for the compact topology on external
+
+exec openshift-install agent create image --dir $WORK
```

Comment:
It would be nice to use isocmp to also check the agent-cluster-install.yaml file in the ISO also contains the external type and external platform name.

Reply:
Done